eos-update-flatpak-repos

```diff
@@ -130,6 +130,33 @@ def _remove_remotes():
                                    '--system', '--force', name])
 
 
+# dict of remote -> URLs to migrate
+REMOTES_TO_MIGRATE = {
+    "eos-runtimes": {
+        "old-url": "https://ostree.endlessm.com/ostree/eos",
+        "new-url": "https://ostree.endlessm.com/ostree/eos-runtimes",
+    },
+}
+
+
+def _migrate_remotes():
+    insts = Flatpak.get_system_installations()
+    inst = insts[0]
+
+    for name, urls in REMOTES_TO_MIGRATE.items():
+        remote = _flatpak_inst_get_remote(inst, name)
+        if remote and remote.get_url() == urls["old-url"]:
+            logging.info("Migrating remote {} from {} to {}"
+                         .format(name, urls["old-url"], urls["new-url"]))
+
+            subprocess.check_call(["flatpak", "remote-modify",
+                                   "--system", "--url", urls["new-url"],
+                                   name])
+            subprocess.check_call(["flatpak", "remote-modify",
+                                   "--system", "--update-metadata",
+                                   name])
```

Comment:
Does this require a network connection? What happens if the computer is offline?

Reply:
Good catch. It does fetch the remote repo config and syncs to it. The important thing you'd lose is not getting the collection ID, so I think I should update the configuration dict so that can be specified here. The only other part would be the title. It appears that flatpak transaction does that automatically, so you'd probably get it later.